modules/ocf_mysql/templates/99-ocf.cnf

```diff
@@ -1,6 +1,10 @@
 [mysqld]
 bind-address = ::
 
+# enable encrypted connections
+ssl-cert=/etc/ssl/private/<%= @fqdn %>.crt
+ssl-key=/etc/ssl/private/<%= @fqdn %>.key
+
```

Comment:
You don't need `ssl-ca` to be specified here too? https://mariadb.com/kb/en/library/securing-connections-for-client-and-server/#configuration